src/Utilities.jl

```diff
@@ -6,11 +6,30 @@ modules in the coupler.
 """
 module Utilities
 
+import Artifacts
 import ClimaComms
 import ClimaCore as CC
 
 export swap_space!
 
+"""
+    artifact_exists(name)
+
+Return whether the artifact with the given name exists and is available to use.
+"""
+function artifact_exists(name)
```

Comment:
some tests for this would be good

Reply:
I cannot test for non-existing artifacts because all the artifacts in the test suite are always automatically downloaded.